makejdk.sh

```diff
@@ -306,6 +306,13 @@ checkInCaseOfDockerShouldTheContainerBePreserved
 setDefaultIfBranchIsNotProvided
 setWorkingDirectoryIfProvided
 setTargetDirectoryIfProvided
-cloneOpenJDKGitRepo
-getOpenJDKUpdateVersion
+time (
+    echo "Cloning OpenJDK git repo"
+    cloneOpenJDKGitRepo
+)
+
+time (
+    echo "Updating the OpenJDK git repo on local environment"
+    getOpenJDKUpdateVersion
```

Comment:
Echo statement is incorrect?  Not sure why you're wrapping a time around this either?